src/test/resources/limetrans/output.fix/metafix-wip.json

```diff
@@ -1,157 +1,327 @@
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ "VARIATIONAL METHODS IN OPTIMUM CONTROL THEORY", "VARIATSIONNYE METODY TEORII OPTIMALNOGO UPRAVLENIYA <ENGL.>" ],
   "TitleOther" : {
     "title" : [ "VARIATIONAL METHODS IN OPTIMUM CONTROL THEORY", "VARIATSIONNYE METODY TEORII OPTIMALNOGO UPRAVLENIYA <ENGL.>" ]
   }
 }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
 {
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ "Tancy małych narodov juga Dal'nego Vostoka SSSR" ],
   "TitleOther" : {
     "title[]" : "Tancy małych narodov juga Dal'nego Vostoka SSSR"
   }
 }
 {
+  "TO" : [ "OEUVRES COMPLETES" ],
   "TitleOther" : {
     "title[]" : "OEUVRES COMPLETES"
   }
 }
 {
+  "TO" : [ "RATIONALISME DANS LA LITTERATURE FRANCAISE DE LA RENAISSANCE" ],
   "TitleOther" : {
     "title[]" : "RATIONALISME DANS LA LITTERATURE FRANCAISE DE LA RENAISSANCE"
   }
 }
-{ }
-{ }
-{ }
-{ }
-{ }
 {
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ "Basics entwerfen materialität" ],
   "TitleOther" : {
     "title[]" : "Basics entwerfen materialität"
   }
 }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
-{ }
 {
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ "Basics Wasserkreislauf im Gebäude", "Basics Haustechnik Wasserkreislauf im Gebäude" ],
   "TitleOther" : {
     "title" : [ "Basics Wasserkreislauf im Gebäude", "Basics Haustechnik Wasserkreislauf im Gebäude" ]
   }
 }
-{ }
-{ }
-{ }
 {
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ ]
+}
+{
+  "TO" : [ "Analytica pro", "EuCheMS newsletter", "Nachrichten für die Karriere", "Nachrichten für die Chemikerkarriere", "Nachrichten aus der Chemiewirtschaft", "Nano", "Mitteilungsblatt der Gesellschaft Österreichischer Chemiker", "Mitteilungsblatt der Gesellschaft Österreichischer Chemiker" ],
   "TitleOther" : {
     "title" : [ "Analytica pro", "EuCheMS newsletter", "Nachrichten für die Karriere", "Nachrichten für die Chemikerkarriere", "Nachrichten aus der Chemiewirtschaft", "Nano", "Mitteilungsblatt der Gesellschaft Österreichischer Chemiker", "Mitteilungsblatt der Gesellschaft Österreichischer Chemiker" ]
   }
 }
 {
+  "TO" : [ "ARH" ],
   "TitleOther" : {
     "title[]" : "ARH"
   }
 }
```

Comment:
This seems to be a good example of the problem of moving an array of strings to an deeper hierachrie level.

Reply:
Yes, indeed. Thanks for pointing it out.